scripts/mkcloud

```diff
@@ -101,7 +112,7 @@ needcvol=1
 : ${xen_node_memory:=4000000}
 [[ "$libvirt_type" = "xen" && $compute_node_memory -lt $xen_node_memory ]] && compute_node_memory=$xen_node_memory
 # hdd size defaults (unless defined otherwise)
-: ${adminnode_hdd_size:=15}
+: ${adminnode_hdd_size:=20}
```

Comment:
ack. we can increase that value by setting the env variable. i will remove that.